x-pack/plugins/lens/public/datasources/form_based/dimension_panel/dimension_editor.tsx

```diff
@@ -665,10 +665,8 @@ export function DimensionEditor(props: DimensionEditorProps) {
                 isOpen={isHelpOpen}
                 display="inlineBlock"
                 panelPaddingSize="none"
-                className="dscFieldTypesHelp__popover"
-                panelClassName="dscFieldTypesHelp__panel"
```

Comment:
For some reason it was using Discover classes which are removed now. I have not noticed any visual changes after cleaning it up.

Reply:
It's for this docs popover:
<img width="494" alt="Screenshot 2023-01-18 at 12 23 12" src="https://user-images.githubusercontent.com/1415710/213161144-f7ae15cc-8e53-41be-a077-b2f0f8ba83e6.png">